File: questions/2924/python/2924.py
from typing import List

class Solution:
    def findChampion(self, n: int, edges: List[List[int]]) -> int:
        strongests = {node for node in range(n)}

        for _, v in edges:
            strongests.discard(v)

        return strongests.pop() if len(strongests) == 1 else - 1

File: questions/2924/cpp/2924.cpp
#include <vector>
#include <set>

class Solution {
public:
    int findChampion(int n, std::vector<std::vector<int>>& edges) {
        std::set<int> strongests = {};

        for(int node = 0; node < n; ++node) {
            strongests.insert(node);
        }

        for (const auto& edge: edges) {
            strongests.erase(edge[1]);
        }

        return strongests.size() == 1 ? *strongests.begin() : -1;
    }
};